programs/test-sndfile-metadata-set.py

```diff
@@ -71,52 +80,52 @@ def make_sine (self):
     def check_executables (self):
         for name in [ self.meta_set_prog, self.meta_get_prog, self.make_sine_prog ]:
             if not (os.path.isfile (name)):
-                print "\n\nError : Can't find executable '%s'. Have you run make?" % name
+                print("\n\nError : Can't find executable '%s'. Have you run make?" % name)
                 sys.exit (1)
 
 
 def print_test_name (name):
-    print "    %-30s :" % name,
+    print("    %-30s :" % name)
 
 def assert_info (programs, filename, arg, value):
     output = programs.meta_get (False, "%s %s" % (arg, filename))
     if output.find (value) < 0:
-        print "\n\nError : not able to find '%s'." % value
-        print output
+        print("\n\nError : not able to find '%s'." % value)
+        print(output)
         sys.exit (1)
```

Comment:
Something about this changes has screwed up the output. Its now:
```
Testing WAV metadata manipulation:
    Empty fail test                :
ok
    Copy test                      :
ok
    Auto date test                 :
ok
    Update test                    :
ok
    Post mod test                  :
ok
    Update test                    :
ok
    Rewrite test                   :
ok
```
The "ok" used to be at the end of the previous line.